mounts_test.go

```diff
@@ -49,6 +50,22 @@ func TestMounts(t *testing.T) {
 			}`),
 		},
 
+		//file mount from raw
+		&FileMount{
+			File: filepath.Join("preloaded", "raw.txt"),
+			Content: json.RawMessage(`{
+				"raw": "Hello Raw Content!"
+			}`),
+		},
+
+		//file mount from base64
+		&FileMount{
+			File: filepath.Join("preloaded", "base64"),
+			Content: json.RawMessage(`{
+				"base64": "ZWNobyAiSGVsbG8gQmFzZTY0ISI="
+			}`),
+		},
```

Comment:
You will need to add two `checkSHA256` at the end of this method (around line 180) to check that the two files got created.

Note, until now all content was downloaded, so we cached it. We technically don't need to cache literal content (raw / base64) but we can change that in another PR, as the current implementation caches all content.